HookCatcher/views.py

```diff
@@ -73,32 +30,63 @@ def stateRepresentation(stateObj):
         'desc': stateObj.state_desc,
         'gitType': stateObj.git_source_type,
         'gitName': stateObj.git_source_name,
-        'gitCommitURL': GITHUB_PR_URL + stateObj.git_commit,
+        'gitCommitSHA': stateObj.git_commit[:7],
         'imgsOfState': stateObj.image_set.all()
     }
 
 
+# only take the information needed from JSON response
+def gitRepresentation(gitJSON):
+    return {
+        'url': gitJSON['html_url'],
+        'author': gitJSON['commit']['author']['name'],
+        'date': gitJSON['commit']['author']['date'],
+        'filesChanged': len(gitJSON['files'])
+    }
+
+
+# get request to github API
+def gitCommit(gitSHA):
+    headers = {
+        'Authorization': 'token ' + settings.GIT_OAUTH,
+    }
+
+    gitRepoURL = os.path.join(settings.GIT_KOLIBRI_API, 'commits')
+    gitCommitURL = os.path.join(gitRepoURL, gitSHA)
```

Comment:
You shouldn't be using `os.path` methods to construct URLs - they are only meant for system paths. On windows, this will give you urls like `http://example.com\commits` :)

See https://docs.python.org/2/library/urlparse.html#urlparse.urljoin